source/Irrlicht/COpenGLTexture.cpp

```diff
@@ -75,8 +75,8 @@ void COpenGLFilterableTexture::regenerateMipMapLevels()
 		return;
 
     COpenGLExtensionHandler::extGlGenerateTextureMipmap(TextureName,this->getOpenGLTextureType());
-}
-
+}
+
 
 uint32_t COpenGLTexture::getOpenGLFormatBpp(const GLenum& colorformat)

```

Comment:
@qbasa12 this one needs switch cases for SRGB as well

P.S. @Crisspl why are there EF_R8_SRGB, EF_R8G8_SRGB.... I'm 100% sure that OpenGL doesn't support that, is that even a thing in Vulkan?